src/Humanizer/InflectorExtensions.cs

```diff
@@ -67,20 +67,25 @@ public static string Titleize(this string input)
         /// By default, pascalize converts strings to UpperCamelCase also removing underscores
         /// </summary>
         /// <param name="input"></param>
+        /// <param name="removeWhiteSpace">"This has a space" -> "ThisHasASpace" if true</param>
         /// <returns></returns>
         public static string Pascalize(this string input)
         {
-            return Regex.Replace(input, "(?:^|_)(.)", match => match.Groups[1].Value.ToUpper());
+            
+            input = Regex.Replace(input, "(?:^|_| )(.)", match => match.Groups[1].Value.ToUpper());
```

Comment:
RegEx isn't my strong suit, but is there a way these two regex statements can be combined into one?

Reply:
I'll have a look what I can come up with